packages/core/src/tools/create-lambda-handler.js

```diff
@@ -203,10 +204,10 @@ const createLambdaHandler = (appRawOrPath) => {
 
           const { skipHttpPatch } = appRaw.flags || {};
           // Adds logging for _all_ kinds of http(s) requests, no matter the library
-          if (!skipHttpPatch) {
+          if (!skipHttpPatch && !event.calledFromCli) {
```

Comment:
I was running `zapier push` and found the CLI tool sent HTTP logs to the log server. So I added `!event.calledFromCli` to make sure we don't patch `http` when it was from CLI.